testing/botan/APKBUILD

```diff
@@ -0,0 +1,41 @@
+# Maintainer: Alexander F Rødseth <xyproto@archlinux.org>
+# Contributor: Angel Velasquez <angvp@archlinux.org>
+# Contributor: Douglas Soares de Andrade <douglas@archlinux.org>
+# Contributor: d'Ronin <daronin@2600.com>
+# Contributor: Hexchain Tong <richard0053@gmail.com>
+# Contributor: Steve Holweg <skytep@gmail.com>
```

Comment:
What are these people? They all has contributed into this simple abuild...?

Reply:
I adapted this file from archlinux, i keep the credits from the source file as i was not alone to make this file